test/functional/sqlsrv/sqlsrv_param_output_variants.phpt

```diff
@@ -25,14 +30,28 @@ function CreateVariantTable($conn, $tableName)
     sqlsrv_free_stmt($stmt);
 }
 
-function TestOutputParam($conn, $tableName)
+function execProcedure($conn, $tsql, $params)
 {
-    // First, create a temporary stored procedure
-    $procName = GetTempProcName('sqlVariant');
+    if (AE\isColEncrypted()) {
+        $stmt = sqlsrv_prepare($conn, $tsql, $params);
+        if ($stmt) {
+            sqlsrv_execute($stmt);
+        }
+    } else {
+        $stmt = sqlsrv_query($conn, $tsql, $params);
+    }
+    return $stmt;
+}
+
+function testOutputParam($conn, $tableName)
+{
+    // First, create a stored procedure
+    $procName = getTempProcName('sqlVariant', false);
```

Comment:
I thought we're not supposed to use temp proc

Reply:
This works even with AE, possibly because it's sql_variants? I can change it.

Reply:
Wait a minute, it's not temporary because of the `false` argument.

Reply:
ohhhhh, then can you just change to getProcName?

Reply:
I'll just change it to not call a common method